### PR TITLE
Run java functional tests on travis

### PR DESCRIPTION
**Pull request by nidhi1605**
Add travis job to run Java functional tests on travis, as per #825 

Signed-off-by: Nidhi Gupta <itsnidhi16@gmail.com>

### CODE REVIEW

File: scripts/build/Dockerfile.openj9-alpine [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	protobuf-c-dev \
+	protobuf-dev \
+	python \
+	sudo \
+	maven
+
+COPY . /criu
+WORKDIR /criu
+
+RUN apk add \
```
Comment (adrianreber) [inline scripts/build/Dockerfile.openj9-alpine]: Why are those packages not installed in the previous run of `apk add`?

Comment (nidhi1605) [reply scripts/build/Dockerfile.openj9-alpine]: I have changed it. All the packages are now installed together.

**Comment by adrianreber** [comment]
Looks good. All tests are passing. Thanks!

File: scripts/travis/travis-tests [hunk 2]
```diff
@@ -170,6 +170,8 @@ make -C test/others/make/ run CC="$CC"
 make -C test/others/shell-job/ run
 make -C test/others/rpc/ run
 
+mvn -f test/javaTests/pom.xml test
```
Comment (avagin) [inline scripts/travis/travis-tests]: I don't think that we need to add this test here. We have two separate travis jobs to run this test and I think these should be enough.

Comment (nidhi1605) [reply scripts/travis/travis-tests]: I have removed the test from travis-tests.sh.

**Comment by avagin** [comment]
Applied, thanks!

**Pull request merged**